tests/testthat/test-assertions.R

```diff
@@ -49,6 +49,9 @@ test_that("has_name works correctly", {
   expect_true(has_name(x, letters[2]))
   expect_false(has_name(x, "something else"))
   expect_false(has_name(x, NA))
+  expect_error({
```

Comment:
Can you please remove the `{}` here, and don't name the second argument.

Reply:
I did this but out of curiosity, what's your reason for not wanting the keyword arg? I feel like getting in that habit is a good way to protect yourself against silent failures. Especially since `expect_error` takes a `...`

Reply:
1. I don't think it makes sense to name the second argument when it's commonly used
2. It would be inconsistent with standard usage of `expect_equal()`
3. It's actually more error prone because a typo like `regex` would cause that argument to be silently ignored